core/README.md

```diff
@@ -16,30 +16,18 @@ Build and deploy the project using one of the following approaches:
 
 Go to the `{luigi-core-root-folder}` folder and run the following command:
 
-`npm run-script bundle && cp -r ./public/ {PATH_TO_SAMPLEAPP}/node_modules/luigi-core/`
+`npm run-script bundle && cp -r ./public/ {PATH_TO_SAMPLEAPP}/node_modules/@kyma-project/luigi-core/`
 
-**Build and deploy in the development mode**
+**Build and deploy in development mode**
 
+For non-minified sources, use this one.
```

Comment:
For non-minified sources, use this approach.

Reply:
done